core/templates/app.constants.ajs.ts

```diff
@@ -20,10 +20,9 @@
 // equivalents of the Angular constants are no longer needed.
 import { AppConstants } from 'app.constants';
 
-import constants from 'assets/constants';
-
-for (var constantName in constants) {
-  angular.module('oppia').constant(constantName, constants[constantName]);
```

Comment:
I think it is fine to keep `import constants from 'assets/constants';` here, so that we do not need to create `commonConstants` in `AppConstants`. But add an explanatory comment.

Reply:
done